### PR TITLE
Tweak license and readme

### PR DESCRIPTION
**Pull request by shaisachs**

### CODE REVIEW

**Comment by joegoldbeck** [comment]
Re: license. Is there an the advantage you're looking for here?

**Comment by shaisachs** [comment]
Fair enough - so you think we should leave no license? Or would you rather a different one?

File: README.md [hunk 1]
```diff
@@ -1,91 +1,7 @@
...
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
-
-
-## Server
```
Comment (joegoldbeck) [inline README.md]: Wait no I needed these. I have cleanup to do there, but these are real commands

File: README.md [hunk 2]
```diff
@@ -1,91 +1,7 @@
...
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
```
Comment (joegoldbeck) [inline README.md]: I suspect some of these are real commands. Perhaps `npm run build` in particular

File: README.md [hunk 3]
```diff
@@ -1,91 +1,7 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Two Cent Stories
 
-## Available Scripts
+The goal of this project is to demonstrate how much we can accomplish with [Elizabeth Warren's Two-Cent Tax](https://elizabethwarren.com/plans/ultra-millionaire-tax). Tell us about your experience with student debt, and see how easily it can be eliminated with the two-cent tax!
```
Comment (joegoldbeck) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (joegoldbeck) [reply README.md]: Personally, I like the readme to allow a new developer to approach the project :)

**Comment by joegoldbeck** [comment]
I've been leaving no license. All rights reserved by default is my understanding.

File: README.md (hunk 1 above)
Comment (shaisachs) [reply README.md]: Oops sorry! I thought that was just boilerplate. I can restore.

**Pull request merged**